recipes/transabyss/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+set -x -e
+
+TOPDIR=`pwd`
+
+mkdir -p $PREFIX/bin
+
+chmod 777 bin/*
+cp bin/* $PREFIX/bin
+cp -rf configs $PREFIX/
+
+mkdir -p $TOPDIR/scripts
+mv transabyss $TOPDIR/scripts/
+mv transabyss-merge $TOPDIR/scripts/
+
+cd $TOPDIR/scripts/
+2to3 -w *
+cd $TOPDIR/utilities
+2to3 -w *
+
+cd $TOPDIR
+cd utilities
+sed -i.bak 's/from utilities import/from transabyss import/' *.py
```

Comment:
I'm not sure what you mean by universal. 

Transabyss is packaged as a set of python scripts, but not as a python package (no setup.py). I want it deployed in conda as a python package, and I don't want for it to include the namespace 'utilities' because that is probably the single most ambiguous namespace of all time.